token-syncer/resources/log4j-test.properties

```diff
@@ -0,0 +1,16 @@
+log4j.rootLogger=DEBUG, file
+
+log4j.appender.file=org.apache.log4j.FileAppender
+log4j.appender.file.File=log/token-syncer-tests.log
+log4j.appender.file.ImmediateFlush=true
+log4j.appender.file.Append=true
+log4j.appender.file.layout=org.apache.log4j.PatternLayout
+log4j.appender.file.layout.conversionPattern=%d{yyyy-MM-dd HH:mm:ss} %-5p [%t] %c{4} - %m%n
+log4j.appender.file.Threshold=DEBUG
+
+# DEBUG is way too noisy for some categories
+log4j.logger.org.apache.http=INFO
```

Comment:
Are all these categories needed?

Reply:
Reduced it to only contain `log4j.logger.org.eclipse.jetty`